desktop/i18n-deprecated/resources.en.json

```diff
@@ -0,0 +1,384 @@
+{
+  "browse-layout.openedAdvancedFilter": "Opened advanced filter blade",
```

Comment:
Update .gitignore to ignore this file, as it's generated at compile-time.

Reply:
Actually all 3 of these lines in .gitignore need to be updated:

```
# Ignore the english compiled file
i18n/resources.en.json
i18n/xliff/
i18n/xliffs/
```

Note you'll need to do a `git rm` to remove this file from source control now that it's added.